api/src/main/resources/workbench-api.yaml

```diff
@@ -5698,6 +5698,12 @@ definitions:
         type: array
         items:
           type: string
+      eligabledAccessTierShortNames:
+        description: Short names of access tiers that user is eligabled. User beomes eligable if their institution allows it
```

Comment:
please change to:

The short names of the access tiers which a user is eligible for.  User eligibility for tiers is based on their institution's tier requirements.  This does not mean the user has access to these tiers; to gain access, the user must still complete all the necessary access requirement steps for each tier.